README.md

```diff
@@ -354,7 +366,7 @@ On the exemple above, the result will be of type `Result<[string, number, boolea
 }
 ```
 
-If one or more of the functions fails, the errors will be concatenated:
+If any of the functions fail, the `errors` field will be populated:
```

Comment:
How to express that the errors of multiple DFs are going to be joined (concatenated)?